speechAssets/IntentSchema.json

```diff
@@ -28,12 +25,6 @@
         "intent": "GetIntroNewsIntent"
       },
       {
-        "intent": "GetIntroReviewsIntent"
-      },
-      {
-        "intent": "GetIntroSportIntent"
```

Comment:
I'm wrong about this - we only trigger this intent internally, so it can be removed from here.